### PR TITLE
Balance wotlk

### PR DESCRIPTION
**Pull request by Ceridwyn**
Balance update for Wotlk.

### CODE REVIEW

File: sim/druid/starfire.go [hunk 1]
```diff
@@ -46,6 +46,10 @@ func (druid *Druid) newStarfireSpell() *core.Spell {
 			}
 		})
 	}
+	// If improved insect swarm and MF active, +3% crit chance
+	if druid.MoonfireDot.IsActive() && druid.Talents.ImprovedInsectSwarm > 0 {
+		effect.BonusSpellCritRating += 0.01 * float64(druid.Talents.ImprovedInsectSwarm)
```
Comment (jimmyt857) [inline sim/druid/starfire.go:51]: `BonusSpellCritRating` needs to be given in terms of crit rating, not crit chance. So you should change this to, `effect.BonusSpellCritRating += 1 * float64(druid.Talents.ImprovedInsectSwarm) * core.CritRatingPerCritChance`

File: sim/druid/wrath.go [hunk 2]
```diff
@@ -31,17 +33,18 @@ func (druid *Druid) registerWrathSpell() {
...
 				druid.applyNaturesSwiftness(cast)
 			},
 		},
 
 		ApplyEffects: core.ApplyEffectFuncDirectDamage(core.SpellEffect{
 			ProcMask:             core.ProcMaskSpellDamage,
-			BonusSpellCritRating: 0,
-			DamageMultiplier:     1 + 0.02*float64(druid.Talents.Moonfury),
+			BonusSpellCritRating: float64(2 * float64(druid.Talents.NaturesMajesty) * 45.91),
+			DamageMultiplier:     1 + 0.02*float64(druid.Talents.Moonfury)*(1+0.01*float64(druid.Talents.ImprovedInsectSwarm)),
```
Comment (jimmyt857) [inline sim/druid/wrath.go:44]: I think you're missing some parenthesis () here. The moonfury term `0.02*float64(druid.Talents.Moonfury)` is not grouped with the `1` at the start so it won't get multiplied correctly.

**Comment by Ceridwyn** [comment]
Issues will be fixed in the next pull request.

**Comment by Gashiraa** [comment]
Closing since we pushed those changes from a project's branch (not from fork)

**Pull request closed without merging**